.travis.yml

```diff
@@ -1,16 +1,17 @@
 language: node_js
+before_script:
```

Comment:
This might be running before the jshint is installed, you could try doing

after_script:
  - "jshint --config=.jshintconfig ."